Makefile

```diff
@@ -215,27 +215,30 @@ dist/img-oci-artifact.tar: target/wasm32-wasip1/$(OPT_PROFILE)/img-oci-artifact.
 	@mkdir -p "dist/"
 	cp "$<" "$@"
 
-load: dist/img.tar
-	sudo ctr -n $(CONTAINERD_NAMESPACE) image import --all-platforms $<
+.PHONY: pull pull-app pull-oci pull-oci-artifact pull-http
+pull: pull-app pull-oci pull-oci-artifact pull-http
+	echo "Pulled all images"
+
+pull-app:
+	ctr image pull ghcr.io/containerd/runwasi/wasi-demo-app:latest
+
+pull-oci:
+	ctr image pull ghcr.io/containerd/runwasi/wasi-demo-oci:latest
+
+pull-oci-artifact:
+	ctr image pull ghcr.io/containerd/runwasi/wasi-demo-oci-artifact:latest
+
+pull-http:
+	ctr image pull ghcr.io/containerd/runwasi/wasi-http:latest
```

Comment:
```suggestion
pull-%:
	ctr image pull ghcr.io/containerd/runwasi/wasi-demo-$*:latest
```

Reply:
ah, this would break wasi-http, as it would translate to wasi-demo-http

Reply:
Perhaps we should just call it `wasi-demo-http` to be consistent with all other images we publish

Reply:
This could be a follow up as I need to republish the images.

Reply:
just created an issue to track: https://github.com/containerd/runwasi/issues/825